scico/linop/_stack.py

```diff
@@ -146,6 +147,83 @@ def _adj(self, y: Union[Array, BlockArray]) -> Union[Array, BlockArray]:  # type
         return snp.blockarray(result)
 
 
+class DiagonalReplicated(DReplicated, LinearOperator):
```

Comment:
I assume you noticed that this is an alias for `operator.DiagonalReplicated`. You don't have similar concerns for `DStack` and `VStack`? If `DReplicated` is changed as suggested, it would seem to make sense to modify them too for consistency.

Reply:
Addressed as discussed.